dashboards-observability/public/components/visualizations/charts/maps/treemap_type.ts

```diff
@@ -49,6 +50,14 @@ export const createTreeMapDefinition = (params: BarTypeParams = {}) => ({
         mapTo: 'dataConfig',
         editor: VizDataPanel,
         sections: [
+          fetchConfigObject('Tooltip', {
```

Comment:
can we move this at some common place as it same for all charts.

Reply:
I have used it like this as in future we can use same helper function for fetching other config objects. In that case "Tooltip", "Legend", "Theme", etc. can be used as the first argument. As we are using this helper function for "Tooltip" only it's looking repetitive.